var/spack/repos/builtin/packages/py-pypar/package.py

```diff
@@ -30,12 +30,13 @@ class PyPypar(Package):
        written in Python to run in parallel on multiple processors and
        communicate using MPI."""
     homepage = "http://code.google.com/p/pypar/"
-    url      = "https://pypar.googlecode.com/files/pypar-2.1.5_108.tgz"
+    url      = "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pypar/pypar-2.1.5_108.tgz"
 
     version('2.1.5_108', '7a1f28327d2a3b679f9455c843d850b8',
-            url='https://pypar.googlecode.com/files/pypar-2.1.5_108.tgz')
+            url = "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pypar/pypar-2.1.5_108.tgz")
```

Comment:
It looks like the URL for the version is the same as the URL. Are you sure the URL for version is necessary?